client/me/profile-links-add-wordpress/index.jsx

```diff
@@ -157,84 +157,94 @@ export default React.createClass( {
 						<Site
 							site={ site }
 							indicator={ false }
-							onSelect={ this.onSelect.bind( this, inputName ) } />
+							onSelect={ this.onSelect( inputName ) } />
 					</li>
 				);
 			} )
 		);
-	},
+	}
 
 	renderAddableSitesForm() {
+		const { translate } = this.props;
 		const checkedCount = this.getCheckedCount();
 
 		return (
 			<form className="profile-links-add-wordpress" onSubmit={ this.onAddableSubmit }>
 				<p>
-					{ this.translate( 'Please select one or more sites to add to your profile.' ) }
+					{ translate( 'Please select one or more sites to add to your profile.' ) }
 				</p>
 				<ul className="profile-links-add-wordpress__list">
 					{ this.renderAddableSites() }
 				</ul>
 				{ this.possiblyRenderError() }
 				<FormButton
 					disabled={ ( 0 === checkedCount ) ? true : false }
-					onClick={ this.recordClickEvent( 'Add WordPress Sites Button' ) }
+					onClick={ recordClickEvent( 'Add WordPress Sites Button' ) }
 				>
-					{ this.translate( 'Add Site', 'Add Sites', { count: checkedCount } ) }
+					{ translate( 'Add Site', 'Add Sites', { count: checkedCount } ) }
 				</FormButton>
 				<FormButton
 					className="profile-links-add-wordpress__cancel"
 					isPrimary={ false }
-					onClick={ this.recordClickEvent( 'Cancel Add WordPress Sites Button', this.onCancel ) }
+					onClick={ recordClickEvent( 'Cancel Add WordPress Sites Button', this.onCancel ) }
 				>
-					{ this.translate( 'Cancel' ) }
+					{ translate( 'Cancel' ) }
 				</FormButton>
 			</form>
 		);
-	},
+	}
 
 	renderInvitationForm() {
+		const { translate } = this.props;
+
 		return (
 			<form>
 				<p>
 					{
-						this.translate(
+						translate(
 							'You have no public sites on WordPress.com yet, but if you like you ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'You have no public sites on WordPress.com yet, but if you like you can create one now.  You may also add self-hosted WordPress sites here after installing {{jetpackLink}}Jetpack{{/jetpackLink}} on them.' )` *ES Score: 18*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).